### PR TITLE
Adding support for django 1.8+

### PR DESCRIPTION
**Pull request by ulyssesv**
Adding support for django 1.8+ TEMPLATES setting in the admin_views_install_templates management command.

### CODE REVIEW

File: admin_views/management/commands/admin_views_install_templates.py [hunk 1]
```diff
@@ -14,7 +16,13 @@ class Command(BaseCommand):
...
     def handle(self, *args, **options):
         current_dir = os.path.dirname(__file__)
-        template_dirs = settings.TEMPLATE_DIRS
+        if StrictVersion(django.get_version()) < StrictVersion('1.8'):
+            template_dirs = settings.TEMPLATE_DIRS
+        else:
+            template_dirs = []
+            for template_engine in settings.TEMPLATES:
+                if template_engine['BACKEND'] == 'django.template.backends.django.DjangoTemplates':
+                    template_dirs = template_dirs + template_engine['DIRS']
```
Comment (keimlink) [inline admin_views/management/commands/admin_views_install_templates.py]: Why not `break` if a template directory for Django's template engine has been found?

**Comment by kostya0shift** [comment]
#21

File: admin_views/management/commands/admin_views_install_templates.py (hunk 1 above)
Comment (jpic) [inline admin_views/management/commands/admin_views_install_templates.py]: +1

**Comment by frankwiles** [comment]
Awesome thanks for the patch!

**Pull request merged**